backend/notes/views.py

```
@@ -1,11 +1,21 @@
 
 from .models import Notebook
 from .serializers import NotebookSerializer
 
-# Create your views here.
+class FetchNotebooksEndpoint(ListAPIView):
+    serializer_class = NotebookSerializer
+    permission_classes = [IsAuthenticated]
+
+    def get_queryset(self):
+        # Filter logic not implemented yet, for future purposes
+        filter = self.request.query_params.getList("filters")
```

Ruff (0.12.2)

Local variable `filter` is assigned to but never used

Remove assignment to unused variable `filter`

(F841)